charts/posthog/README.md

```diff
@@ -16,47 +16,129 @@ This chart bootstraps a [PostHog](https://posthog.com/) deployment on a [Kuberne
 
 ## Prerequisites
 
-- [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
 - [Kubernetes](https://kubernetes.io/) 1.6+ with Beta APIs enabled
 - [Helm](https://helm.sh/) >= v3
 
+## Setting up Kuberentes Cluster
+
+<details>
+  <summary>
+    <b>Google Cloud</b>
+  </summary>
+<br />
+See [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
+</details>
+
+<details>
+  <summary>
+    <b>AWS</b>
+  </summary>
+<br />
+TODO
+</details>
+
+<details>
+  <summary>
+    <b>Digital Ocean</b>
+  </summary>
+<br />
+Note that there is a [one click](todo: link) option to deploy Posthog on DigitalOcean
+</details>
+
 ## Installing the chart
 
-To install the chart with the release name `posthog`, run the following:
+To install the chart with the release name `posthog` in `posthog` namespace, run the following:
 
 ```console
 helm repo add posthog https://posthog.github.io/charts-clickhouse/
 helm repo update
-helm install -f values.yaml --timeout 20m posthog posthog/posthog
+helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
+```
+
+Where the `values.yaml` has the following minimum content:
```

Comment:
This structure is kind of bad. We now have install section which gives some overview of configuration and then a separate configuration section.

I'd recommend keeping all of configuration in one place and _linking_ to relevant sections instead. As-is this just makes the flow harder to follow.

Reply:
The way I saw it is "Configuration" is optional stuff, this in here is mandatory (so the user needing to go down to the configuration section added complexity for most folks, who just wanted to make the simplest thing run which I believe is the majority). I'd love if we didn't need it, but I haven't seen a way we could make the chart work with default values for multiple deployments. I looked into it and then only needing 1 value (cloud being the only required setting through values.yaml or `--set cloud=gcp` or `--set cloud=aws`), but being able to enable/disable nginx,certManager was tricky based on other values in the yaml sadly.